ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1875,6 +1868,30 @@ AP_ServoRelayEvents *GCS_MAVLINK_Copter::get_servorelayevents() const
     return &copter.ServoRelayEvents;
 }
 
+#if ADVANCED_FAILSAFE == ENABLED
+AP_AdvancedFailsafe *GCS_MAVLINK_Copter::get_advanced_failsafe() const
+{
+    return &copter.g2.afs;
+}
+#endif
+
+MAV_RESULT GCS_MAVLINK_Copter::handle_flight_termination(const mavlink_command_long_t &packet) {
+    MAV_RESULT result = MAV_RESULT_FAILED;
+
+#if ADVANCED_FAILSAFE == ENABLED
+    if (super(packet) != MAV_RESULT_ACCEPTED) {
```

Comment:
Well, I would say then that the disarm motors part should always be done (provided the param1 is bigger than 0.5), independently of AFS returning value.

By the way, if AFS returns accepted this will return failed, the _result_ isn't set.